pandas/tests/generic/test_finalize.py

```diff
@@ -227,12 +227,10 @@
         (pd.DataFrame, frame_data, operator.methodcaller("nunique")),
     ),
     pytest.param(
-        (pd.DataFrame, frame_data, operator.methodcaller("idxmin")),
-        marks=not_implemented_mark,
+        (pd.DataFrame, frame_data, operator.methodcaller("idxmin"))
     ),
     pytest.param(
-        (pd.DataFrame, frame_data, operator.methodcaller("idxmax")),
-        marks=not_implemented_mark,
+        (pd.DataFrame, frame_data, operator.methodcaller("idxmax"))
```

Comment:
I think you can also remove the `pytest.param` call here now that the mark is not added

Reply:
Noted. Thanks!